sdks/core-api-sdk/src/sdks/AwsS3SDK/index.ts

```diff
@@ -0,0 +1,48 @@
+import { HttpRequestMethods } from '@ultras/services/NetworkService/types';
+import CoreApiBaseSDK, { Mode } from '../CoreApiBaseSDK';
+import { DynamicQueryParam } from '../types';
+import {
+  SigningUrlParamsInterface,
+  UploadViaSignedUrlParamsInterface,
+  UploadParamsInterface,
+} from './types';
+
+export class AwsS3SDK extends CoreApiBaseSDK {
```

Comment:
Please change the name of this, so mention this is `Ultras` or `Custom`, there is an sdk for official s3 service

Reply:
did you mean `UltrasS3SDK`? we can use `FileUploadS3SDK`